.github/workflows/java-microservice-ci-default.yml

```diff
@@ -342,6 +342,36 @@ jobs:
       - name: Helm Package
         run: helm package ${{ inputs.project_name }}-service/target/helm/${{ inputs.project_name }} -d ${{ inputs.project_name }}-service/target/helm/
 
+      # Docker SBOM
+      - name: Generate sbom docker db
+        uses: anchore/sbom-action@v0.15.9
+        if: ${{ inputs.enable_database_docker }}
+        with:
+          image: bruxless/${{ inputs.project_name }}-db-${{ steps.get_version.outputs.project_version }}
+          format: cyclonedx
```

Comment:
why not using json ? or better use both and generate at with the two format ?
I had trouble using xml with some tools that only accept json ...

Reply:
Yes, i will switch to json